css-media-queries/meta-viewport.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <!-- your code here 👇 -->
+    <meta name="viewport" />
```

Comment:
almost! you want to make sure to have the entire tag: 
` <meta name="viewport" content="width=device-width, initial-scale=1.0">`